Makefile

```diff
@@ -176,12 +176,12 @@ cluster-create::
 
 .PHONY: dev-install
 dev-install::
-	$(SCRIPTS_DIR)/fetch kind 0.17.0
+	$(SCRIPTS_DIR)/fetch kind 0.23.0
```

Comment:
would it be better to have a variable that holds this value? Update would require changing only it instead of the places that need it